### PR TITLE
Css

### PR DESCRIPTION
**Pull request by Gwenwyfar**
Just some more details fixed/improved. A lot of edits, but these are mostly minor. Just getting these in before we start having conflicts.

### CODE REVIEW

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -7,14 +7,12 @@ body {
...
-	margin: 0;
-	padding: 0;
 	display: flex;
 	flex-direction: column;
 	min-height: 100vh;
 }
-/* <p> is a possible exception for margins. Must test that one a bit. */
-p, ul, ol, li, dl, dd ,dt, fieldset, form {
+/* General reset */
+* {
```
Comment (XinYenFon) [inline Themes/default/css/index.css:15]: Well this is an important step, not that I'm against but requires some careful testing to be safe.

File: Themes/default/css/index.css [hunk 2]
```diff
@@ -392,12 +385,12 @@ blockquote cite::before {
...
 }
 /* Styling for BBC tags */
-.bbc_link:link, .bbc_link:visited {
+.bbc_link {
 	border-bottom: 1px solid #A8B6CF;
 }
 .bbc_link:hover {
 	text-decoration: none;
-	border-bottom: 1px solid #346;
+	border-bottom-color: #346;
```
Comment (XinYenFon) [inline Themes/default/css/index.css]: changes like this requires consistency, either go for all in CSS or just revert it.

**Review by XinYenFon — CHANGES_REQUESTED** [review]
Rest looks fine.

File: Themes/default/css/index.css (hunk 1 above)
Comment (Gwenwyfar) [reply Themes/default/css/index.css:15]: Margin/padding should be fine, browser default ones are too inconsistent to be relied onto. Border-box could cause some harm, but at most it'll break somewhere that widths are used that conflict with padding, better to find those early and they're very easy to fix. I looked at a wide range of pages and didn't find any problems so far, other than the two already fixed.

File: Themes/default/css/index.css (hunk 2 above)
Comment (Gwenwyfar) [reply Themes/default/css/index.css]: Let's leave it like that for now then.

**Review by XinYenFon — APPROVED** [review]

File: Themes/default/css/index.css (hunk 1 above)
Comment (XinYenFon) [reply Themes/default/css/index.css:15]: Those type of padding issues can be fixed in RC period.

**Review by XinYenFon — APPROVED** [review]

**Review by frandominguezl — APPROVED** [review]

**Pull request merged**